Design Layout implemented

afvc opened the pull request:
- New design structure
- new sass files (more division)
- Far hide and show with focus
- Login and Sign Up merged on one page with modals instead of separated html
- New box page “deleted” and changed into a modal
- Settings extra text boxes with same logic as far
- 90% flex box -grid only flex box on box_list
- all website static in the moment
- responsiveness is resolved in principle
-  #mobile-first

afvc committed:
first design changes

afvc committed:
more changes

afvc committed:
Responsiveness 80% complete on landing page. Sass files separated for better readability

MariCerv committed:
Text sizes and boxes

afvc committed:
more responsivness changes in landing

afvc committed:
Merge

afvc committed:
mobile-first! Mobile works 95%, Medium size still has some problems at the how section.

MariCerv committed:
Boxes

MariCerv committed:
Merge

MariCerv committed:
last alterations yesterday

afvc committed:
landing 99%

MariCerv committed:
Boxes Organization

MariCerv committed:
Merge pull request #82 from whitesmith/teste-mariana

Boxes Organization

afvc committed:
Landing responsive except for navbar

It seems that the landing page is 100% responsive. #mobile-first.
A subtitle map font-size was created. Other maps sizes where reviewed
and changed.

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

# Conflicts:
#	hawkpost/static/css/style.css

afvc committed:
Merge pull request #83 from whitesmith/ux-teste-filipa

Ux teste filipa

afvc committed:
box-buttons solved (close+delete)

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Boxes flexboxgrid

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
button delete + close resolvido

MariCerv committed:
Merge pull request #85 from whitesmith/teste-mariana

Button delete + close resolvido

afvc committed:
landing stuff

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

# Conflicts:
#	boxes/templates/boxes/box_list.html
#	hawkpost/static/css/style.css
#	stylesheets/_forms.scss

afvc committed:
Merge pull request #86 from whitesmith/ux-teste-filipa

Ux teste filipa

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Buttons fixed + new open button

MariCerv committed:
Merge pull request #87 from whitesmith/teste-mariana

Buttons fixed + new open button

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
landing text + how section sizes  fix

afvc committed:
merge

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
align and font size fixed

Fixed buttons
Added a new box button
Align and font size fixed

MariCerv committed:
Merge pull request #88 from whitesmith/teste-mariana

Align and font size fixed

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Landing responsiveness complete

afvc committed:
Responsive navbar 50%. Needs button on mobile to open/close

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
Merge pull request #89 from whitesmith/ux-teste-filipa

Navbar + landing complete

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Landing page complete (responsive 100%)

afvc committed:
nabber complete (100% responsive) + new sass file (_mobileNavigation)

afvc committed:
Merge pull request #90 from whitesmith/ux-teste-filipa

navbar + landing 100% responsive

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Box list final

MariCerv committed:
Merge branch 'teste-mariana' into design_improvements

pjaneiro committed:
Merge

MariCerv committed:
Merge remote-tracking branch 'origin/design_improvements' into design_improvements

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Mini fix on mobile menu

Commit:
Merge pull request #92 from whitesmith/teste-mariana

Mini fix on mobile menu

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Mini fix button boxes

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
modal initialization

MariCerv committed:
Merge pull request #93 from whitesmith/teste-mariana

modal initialization

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Modal improvement

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
modal experiments

afvc committed:
Merge pull request #94 from whitesmith/ux-teste-filipa

modal experiments

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
modal responsive + formatted

afvc committed:
Merge pull request #95 from whitesmith/ux-teste-filipa

modal responsive

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
added mixins + corrected responsiveness error on modal

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Fixes + monitor test

MariCerv committed:
Merge pull request #96 from whitesmith/teste-mariana

fixes + monitor testing

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
login modal + changed media queries to breakpoints

afvc committed:
Merge pull request #97 from whitesmith/ux-teste-filipa

modal login

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
breakpoint changes + login modal

MariCerv committed:
modal fixes

MariCerv committed:
Merge pull request #98 from whitesmith/teste-mariana

modal fixes

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
modal xs + login and sign in on same page

MariCerv committed:
Modal fixes 2

MariCerv committed:
Merge pull request #99 from whitesmith/teste-mariana

modal fixes 2

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
Merge pull request #100 from whitesmith/ux-teste-filipa

modal xs

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
Modal completed

afvc committed:
Merge pull request #101 from whitesmith/ux-teste-filipa

modal completed

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
new settings layout  + sass files divided

afvc committed:
font size correction

afvc committed:
Merge pull request #102 from whitesmith/ux-teste-filipa

final layout imporvs

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

pjaneiro committed:
Fixed bug with modal close button

pjaneiro committed:
Merge

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

dethos commented on boxes/templates/boxes/box_create.html:
```diff
@@ -19,7 +19,7 @@ <h2 class="text-dark">Create New Box</h2>
                 <p>
                     {{ form.description.errors }}
-                    {{ form.description.label_tag }}
+                    {{ form.descyription.label_tag }}
```
Typo here

afvc replied:
eu tenho quase a certeza que isso já foi resolvido, no meu html isso não aparece

afvc replied:
btw, o meu github for mac tem as cores trocadas do que aparentemente vocês costumam ter, não sei se afetará tb aqui no site.

dethos replied:
You might have not pushed the last changes to this github branch

afvc replied:
já vou ligar o servidor nesse branch e confirmar, mas tenho ideia que sim, tudo o resto que alterei estava lá já.

afvc replied:
sim, está tudo dentro dos conformes, tanto no site como no código. dunno
<img width="819" alt="captura de ecra 2016-08-19 as 12 41 19" src="https://cloud.githubusercontent.com/assets/19707794/17808863/79a868a4-660a-11e6-8b60-f65c7bca4a30.png">

dethos replied:
You are looking at the wrong file. This one is `box_create.html`.

dethos commented on boxes/templates/boxes/box_list.html:
```diff
@@ -1,81 +1,189 @@
+<script src="{% static 'javascripts/box_create.js' %}"></script>
+
+
+</section>
```
This section shouldn't be ended inside the `script`'s block. Probably you should end it after line 176, before ending the `content` block.

afvc replied:
ok, will do both changes and then do i do a new pull request or how do i proceed?

dethos replied:
Just push the changes to this branch and they will end up here.

afvc committed:
typo + section closed on wrong place --> solved

dethos commented on stylesheets/_modal-cópia.scss:
```diff
@@ -0,0 +1,90 @@
+     form {
```
please keep the file names in English. based on the name this looks like a duplicated file.

afvc replied:
that file probably is an error. i do not have that in the style.scss being called. i do have _modal 
and _modalxs. in principle you can delete it

dethos replied:
ok :+1: , please do it and push the changes.

afvc committed:
error deleted

dethos committed:
fixed conflicts

afvc committed:
box submit new layout applied

afvc committed:
Merge pull request #104 from whitesmith/ux-teste-filipa

box submit

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Box list fixed

MariCerv committed:
Merge pull request #106 from whitesmith/teste-mariana

Box list fixed

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
forgotten pages

afvc committed:
box submit + password reset corrected

MariCerv committed:
Merge pull request #107 from whitesmith/teste-mariana

forgotten pages

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
Merge pull request #108 from whitesmith/ux-teste-filipa

bos submit

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
erros correction on modals and settings

afvc committed:
Merge pull request #109 from whitesmith/ux-teste-filipa

modal errors + settings

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
general color change + modal rearrangements

afvc committed:
Merge pull request #110 from whitesmith/ux-teste-filipa

color change

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
page color divided

afvc committed:
Merge pull request #111 from whitesmith/ux-teste-filipa

color divisions

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
color changes general

afvc committed:
Merge pull request #112 from whitesmith/ux-teste-filipa

color changes

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
login + sign in modals layout renewed (smaller) - Modals xs size

modal color change
overlay color change
modal text defined in css class because html was not being recognized
in all p tags
details class u-blocker updated with a little margin-top:10px
better alignment and organization in base to understand sections and
colors

afvc committed:
Merge pull request #113 from whitesmith/ux-teste-filipa

modal xs finished

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
box list color correction

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Merge pull request #114 from whitesmith/teste-mariana

Box list color correction

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
input good practices

afvc committed:
t :Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
Merge pull request #115 from whitesmith/ux-teste-filipa

good practices

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
box list final colors

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Merge pull request #116 from whitesmith/teste-mariana

box list final colors

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

afvc committed:
box modal solved. turned into modal xs

afvc committed:
Merge pull request #117 from whitesmith/ux-teste-filipa

boxes modal corrected

afvc committed:
Merge branch 'design_improvements' into ux-teste-filipa

MariCerv committed:
success and close pages color

MariCerv committed:
Merge pull request #118 from whitesmith/teste-mariana

success and close pages color

MariCerv committed:
Merge branch 'design_improvements' into teste-mariana

MariCerv committed:
Color + minor fixes

MariCerv committed:
Merge pull request #120 from whitesmith/teste-mariana

color + minor fixes

afvc committed:
nabber links changed + modal buttons on landing

afvc committed:
navbar links

afvc committed:
nabber support links to help.html

afvc committed:
Merge pull request #121 from whitesmith/ux-teste-filipa

navbar links changed

afvc committed:
Merge remote-tracking branch 'origin/design_improvements' into design_improvements
